backend/controllers/prisonerProfile/prisonerVisits.ts

```diff
@@ -13,9 +13,9 @@ const sortByListSequenceThenDescription = (left, right): number => {
   return listSeqSort !== 0 ? listSeqSort : compareStrings(left.description, right.description)
 }
 
-const sortByLeadThenAgeThenLastNameFirstName = (left, right): number => {
-  if (right.leadVisitor) return 1
-  if (left.leadVisitor) return -1
+const sortByLeadIf18OrOverThenAgeThenLastNameFirstName = (left, right): number => {
+  if (right.leadVisitor && right.age >= 18) return 1
```

Comment:
according to policy shouldn't get any lead visitors under 18, but need to ensure they appear sorted in the children section just in cased

Reply:
I remember Nev saying that in practice children often appear as lead visitors